src/logging.h

```diff
@@ -10,7 +10,9 @@
 #include <boost/log/common.hpp>
 #include <boost/log/sinks.hpp>
 
-extern boost::shared_ptr<boost::log::sinks::asynchronous_sink<boost::log::sinks::text_ostream_backend>> sink;
+// local includes
+#include <src/utility.h>
```

Comment:
This is my final question. Is this include needed?

Reply:
Nope, I don't think so.